melodic/distribution.yaml

```diff
@@ -4687,6 +4687,26 @@ repositories:
       type: git
       url: https://github.com/inertialsense/inertial_sense_ros.git
       version: master
+  inno_sim_interface:
+    doc:
+      type: git
+      url: https://github.com/InnopolisAero/inno_sim_interface.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/InnopolisAero/inno_sim_interface.git
```

Comment:
@frontw please add a LICENSE file at the root of the repository and update its package.xml, which still shows `<license>TODO</license>` for license metadata.  

I've also noticed that dependencies declared in the package.xml do not match dependencies actually used either by the CMakeLists.txt or the [`innosim_relay.launch`](https://github.com/InnopolisAero/inno_sim_interface/blob/master/launch/innosim_relay.launch) file. Consider cleaning things up a bit.